press/saas/doctype/product_trial_request/product_trial_request.py

```diff
@@ -390,3 +391,97 @@ def expire_long_pending_trial_requests():
 		"Expired",
 		update_modified=False,
 	)
+
+
+def gather_stats(time_ago):
+	stats = {
+		"total_trial": 0,
+		"failed_trials": 0,
+		"succeeded_trials": 0,
+		"expired_trials": 0,
+		"pending_trials": 0,
+		"app_wise_failures": {},
+		"total_creation_time": 0,
+		"valid_trials_with_timing": 0,
+	}
+	try:
+		trial_requests = frappe.db.get_all(
+			"Product Trial Request",
+			{"creation": (">", time_ago), "owner": ("not like", "fc-signup-test_%")},
```

Comment:
Check once, If `Product Trial Request` has index on `creation` field. 
Framework, by default index the `modified` field.